Containerfile

```diff
@@ -0,0 +1,17 @@
+FROM fedora:40
+RUN dnf install -y fastapi-cli python3-fastapi python3-requests python3-drain3 python3-pip python3-pydantic-settings python3-starlette+full \
+    && pip3 install sse-starlette starlette-context huggingface_hub[cli] \
+    && mkdir /src
+
+# we need to bind mount models: this takes a lot of time to download and makes the image huge
+RUN mkdir /models \
+    && huggingface-cli download TheBloke/Mistral-7B-Instruct-v0.2-GGUF mistral-7b-instruct-v0.2.Q4_K_M.gguf --local-dir /models --local-dir-use-symlinks False
```

Comment:
If we were distributing this, one could argue for a benefit of packaging both runtime and model together, like llamafile does it. But in this case we should just drop the whole layer and make a mount instead.